frontend/device/pocketbook/device.lua

```diff
@@ -238,6 +247,8 @@ if codename == "PocketBook 840" then
     return PocketBook840
 elseif codename == "PB631" then
     return PocketBook631
+elseif codename == "PB627" then
```

Comment:
Could you put it above the 631? (Alpabetical/numerical order)

Reply:
done